drivers/opencl/src/main/java/uk/ac/manchester/tornado/drivers/opencl/runtime/OCLCompilableTask.java

```diff
@@ -48,7 +48,11 @@
     private final Map<OCLBackend, OCLInstalledCode> codeCache;
 
     public OCLCompilableTask(ScheduleMetaData meta, String id, Method method, Object thisObject, Object... args) {
-        super(meta, id, method, thisObject, args);
+        this(meta, id, Collections.emptyMap(), method, thisObject, args);
+    }
+    
+    public OCLCompilableTask(ScheduleMetaData meta, String id, Map<String, Object> properties, Method method, Object thisObject, Object... args) {
+        super(meta, id, properties, method, thisObject, args);
```

Comment:
What are these properties for?

Reply:
`OCLCompilableTask extends CompilableTask` so these properties are programmatic properties passed via CompilableTask API to the superclass. Btw, `OCLCompilableTask ` is never used.